### PR TITLE
Accepted to db

### PR DESCRIPTION
**Pull request by simaoalicechen**
Move the schoolcard that has been accepted to a list to be stored in db. This pr only has all the frontend elements and they work as expected.

### CODE REVIEW

File: src/frontend/shortlist/src/App.vue [hunk 1]
```diff
@@ -10,6 +10,22 @@ const apiClient = new ShortlistApi("https://api.shortlist.nyc");
 const router = useRouter();
 const appSessionStore = sessionStore();
 
+function moveAcceptedSchoolCardToDB(payload) {
```
Comment (adelaidemz) [inline src/frontend/shortlist/src/App.vue:13]: Can this be markSchoolAsAccepted instead?
```suggestion
function markSchoolAsAccepted(payload) {
```

File: src/frontend/shortlist/src/App.vue [hunk 2]
```diff
@@ -10,6 +10,22 @@ const apiClient = new ShortlistApi("https://api.shortlist.nyc");
...
+  console.log(payload);
+  let apiReq = {
+    recoID: payload.recoID,
+    accepted: payload.accepted,
+  };
+  let success = () => {
+    console.log("Added to a list successfully");
```
Comment (adelaidemz) [inline src/frontend/shortlist/src/App.vue:20]: This doesn't actually add to the list, only marks the recommendation as accepted. Same for error message below

File: src/frontend/shortlist/src/api/shortlist.js [hunk 3]
```diff
@@ -36,10 +36,37 @@ export default class ShortlistApi {
...
+    this.recoID = payload.recoID;
+    this.accepted = payload.accepted;
+    this.successCb = successCb;
+    this.failCb = failCb;
+  }
+  execute() {
+    // Send the accepted card first
+    axios({
+      method: "POST",
+      url: "https://api.shortlist.nyc",
```
Comment (adelaidemz) [inline src/frontend/shortlist/src/api/shortlist.js:58]:
```suggestion
      url: "https://api.shortlist.nyc/recommendation/update",
```

File: src/frontend/shortlist/src/App.vue (hunk 2 above)
Comment (simaoalicechen) [reply src/frontend/shortlist/src/App.vue:20]: What is the error message you were referring to? Anything from the console?

**Comment by simaoalicechen** [comment]
Will create another pr.

**Pull request closed without merging**